contrib/packs/actions/pack_mgmt/setup_virtualenv.py

```diff
@@ -98,9 +98,15 @@ def _setup_pack_virtualenv(self, pack_name):
                           (pack_name, virtualenv_path))
 
     def _create_virtualenv(self, virtualenv_path):
-        self.logger.debug('Creating virtualenv in "%s"' % (virtualenv_path))
```

Comment:
Little strange to see config defined in st2actions being used in an action. Considering a pack can itself have config this is somewhat strange.